src/main/kotlin/com/openlattice/chronicle/services/download/DataDownloadService.kt

```diff
@@ -122,13 +185,37 @@ class DataDownloadService(
         return PostgresDownloadWrapper(pgIter).withColumnAdvice(CHRONICLE_USAGE_EVENTS.columns.map { it.name })
     }
 
-    private fun getSensorDataHelper(
+    override fun getParticipantData(
         studyId: UUID,
         participantId: String,
-        sensors: Set<SensorType>
+        dataType: ParticipantDataType,
+        token: String
     ): Iterable<Map<String, Any>> {
+        return getParticipantDataHelper(
+            studyId,
+            participantId,
+            dataType
+        )
+    }
+
+    override fun getParticipantsSensorData(
+        studyId: UUID,
+        participantIds: Set<String>,
+        sensors: Set<SensorType>,
+        startDateTime: OffsetDateTime?,
+        endDateTime: OffsetDateTime?
+    ): Iterable<Map<String, Any>> {
+        if (sensors.isEmpty()) {
+            logger.warn(
+                "study does not have any configured sensors" + ChronicleServerUtil.STUDY_PARTICIPANTS,
+                studyId,
+                participantIds
+            )
+            return listOf()
+        }
+
         val (_, hds) = storageResolver.resolveAndGetFlavor(studyId)
-        val colsAndSql = getSensorDataColsAndSql(sensors)
+        val colsAndSql = getSensorDataColsAndSql(sensors, startDateTime, endDateTime)
```

Comment:
A better pattern here is to have the controller substitute default values of OffsetDateTime.MIN and OffsetDateTime.MAX and only have one prepared statement.